demo/stun/test-server/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"net"
+
+	"github.com/scionproto/scion/pkg/snet"
+)
+
+func main() {
+	var localAddr snet.UDPAddr
+	flag.Var(&localAddr, "local", "Local address")
+	flag.Parse()
+
+	conn, err := net.ListenUDP("udp", localAddr.Host)
+	if err != nil {
+		log.Fatalf("Failed to listen on UDP connection: %v\n", err)
+	}
+	defer conn.Close()
+
+	log.Println("Server running")
```

Comment:
log.Print("Server running")

For consistency, put this statement at the same place as `log.Println("Client running")`.